### PR TITLE
Making test for KeyboardEvent work with jQuery

### PR DESCRIPTION
**Pull request by phillipskevin**
jQuery calls `event.hasOwnProperty('type')` which fails and ends up
throwing an error with KeyboardEvents.

closes https://github.com/canjs/can-jquery/issues/85.

### CODE REVIEW

**Comment by bmomberger-bitovi** [comment]
Make sure you test IE with any fixes for this.  I was going to submit a PR directly to jQuery about this and then couldn't make it work across all platforms.

**Comment by phillipskevin** [comment]
I ended up reverting this change and fixing it in can-jquery. I'll make sure that works in IE. Thanks.

File: dom/events/enter/enter-test.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ function makeKeyboardEvent() {
 		event = new KeyboardEvent("keyup", {
 			key: "Enter"
 		});
+		Object.defineProperty(event, "type", {
```
Comment (justinbmeyer) [inline dom/events/enter/enter-test.js:17]: why is this needed?

Comment (bmomberger-bitovi) [reply dom/events/enter/enter-test.js:17]: The type property on native Events isn't enumerable on some platforms.

Comment (phillipskevin) [reply dom/events/enter/enter-test.js:17]: jquery checks `event.hasOwnProperty("type")`. If that fails, it does `event.indexOf(...)` which throws an error with KeyboardEvents. Anyway, I reverted this change and moved it into can-jquery: https://github.com/canjs/can-jquery/pull/86/files.

Comment (justinbmeyer) [reply dom/events/enter/enter-test.js:17]: I thought this is what John and I fixed.  I made sure that jQuery was passed a copied object.
